frontend/packages/console-shared/src/selectors/common.ts

```diff
@@ -1,10 +1,13 @@
 import * as _ from 'lodash';
-import { K8sResourceCommon } from '@console/internal/module/k8s';
+import { K8sResourceCommon, NodeKind } from '@console/internal/module/k8s';
 
 export const hasLabel = (obj: K8sResourceCommon, label: string): boolean =>
   _.has(obj, ['metadata', 'labels', label]);
 export const getName = <A extends K8sResourceCommon = K8sResourceCommon>(value: A) =>
   _.get(value, 'metadata.name') as K8sResourceCommon['metadata']['name'];
+export const getNodeArchitecture = (node: NodeKind) => {
+  return node?.status?.nodeInfo?.architecture;
+};
```

Comment:
Also, it should have a return type and return an empty string by default:
```suggestion
export const getNodeArchitecture = (node: NodeKind): string => {
  return node?.status?.nodeInfo?.architecture ?? '';
};
```

Reply:
Thanks , done with modification . please review .